examples/Math_Test.eve

```diff
@@ -0,0 +1,273 @@
+# Math Module
```

Comment:
Lol, that wasn't deliberate, I'd added the file to my own repo and it automatically added itself when I pushed the clean lines commit.

Do you want to reject the eve file or should I wind the commits back?